UPGRADE_FROM_FACTORY_GIRL.md

```diff
@@ -35,7 +35,7 @@ to replace all references with the new constant should do the trick. For
 example, on OS X:
 
 ```sh
-grep -e FactoryGirl **/*.rake **/*.rb -s -l | xargs sed -i "" "s|FactoryGirl|FactoryBot|"
+grep -rl -e FactoryGirl --include="*.rake" --include="*.rb" . | xargs sed -i "" "s|FactoryGirl|FactoryBot|"
```

Comment:
The include could also be `--include=\*.{rb,rake}` what do you think?

Reply:
That works as well.

Reply:
This removes the `-s` flag in the first but not the second command. It should probably stay.